.asf.yaml

```diff
@@ -38,11 +38,15 @@ github:
   protected_branches:
     main:
       required_pull_request_reviews:
+        # if we want to use this, GitHub CODEOWNERS should be cleanly define, with different patterns/owners
```

Comment:
I'd be ok with identifying a few critical areas where specific people need to approve as long as PMC members can override in case the review is delayed beyond agreed deadlines.

I'm not sure if we'd want to assign reviewer for every file, though.

Reply:
This is an interesting proposal we might consider for certain features/areas.